src/baseball/main.py

```
@@ -1,10 +1,58 @@
+    return player_number
+
+
+def game_comp(com_number, player_number) :
+    strike, ball, index = 0, 0, 0
+  
+    for i in com_number :
+        if int(player_number[index]) == i :
+            strike += 1
+        elif int(player_number[index]) in com_number :
+            ball += 1
+        index += 1
```

Ruff (0.8.2)

Use `enumerate()` for index variable `index` in `for` loop

(SIM113)